crates/binjs_generate_library/src/lib.rs

```diff
@@ -963,10 +970,23 @@ impl<R> Deserialization<R, Option<{name}>> for Deserializer<R> where R: TokenRea
                     null = null_name,
                     lowercase_name = name.to_rust_identifier_case()
                         .trim_right_matches('_'),
-                    fields = interface.contents()
+                    fields_def = interface.contents()
+                        .fields()
+                        .iter()
+                        .map(|field| format!("
+        dump_format!(self.reader, \".{name}\");
+        let data_{name} = (self.deserialize() as Result<{spec}, R::Error>)?;
+",
+                            name = field.name().to_rust_identifier_case(),
+                            spec = match field_specs.get(field.name()) {
+                                Some(s) => s,
+                                None => "???"
```

Comment:
Could you document that `"???"`?

Reply:
I'll replace it with unwrap() call, which I learned just yesterday :)